src/calendar/internal.tsx

```diff
@@ -134,7 +137,7 @@ export default function Calendar({
     baseDate,
     focusableDate,
     granularity,
-    isDateEnabled,
+    isDateEnabled: isDateFocusable,
```

Comment:
Maybe `isDateEnabled` in `useCalendarGridKeyboardNavigation` should be renamed to `isDateFocusable`?